features/support/user_cleanup.rb

```diff
@@ -0,0 +1,5 @@
+After do
+  if User.present?
+    User.last.destroy
```

Comment:
Question, why call it on the class? It'll just return true if there's any user present at all. Should you be deleting "all" the users if its for cleanup?